service/role.go

```diff
@@ -339,6 +363,15 @@ func (r *roleService) updateRoleToken(ctx context.Context, domain, role, proxyFo
 	return rt.(*RoleToken), err
 }
 
+func roleCacheMemoryUsage(data *cacheData) int64 {
+	structSize := int64(unsafe.Sizeof(data.token) + unsafe.Sizeof(data.domain) + unsafe.Sizeof(data.role) + unsafe.Sizeof(data.proxyForPrincipal) + unsafe.Sizeof(data.minExpiry) + unsafe.Sizeof(data.maxExpiry))
+	stringSize := int64(len(data.domain) + len(data.role) + len(data.proxyForPrincipal))
+	rtStructSize := int64(unsafe.Sizeof(data.token.Token) + unsafe.Sizeof(data.token.ExpiryTime))
+	rtStringSize := int64(len(data.token.Token) + len(strconv.FormatInt(data.token.ExpiryTime, 10)))
+
+	return structSize + stringSize + rtStructSize + rtStringSize
+}
```

Comment:
1. simplify sizeOf(struct)
2. remove `len(strconv.FormatInt(data.token.ExpiryTime, 10)))`

example: https://go.dev/play/p/xSTVoHXIe47
```suggestion
func roleCacheMemoryUsage(data *cacheData) int64 {
	structSize := int64(unsafe.Sizeof(*data))
	stringSize := int64(len(data.domain) + len(data.role) + len(data.proxyForPrincipal))
	if data.token == nil {
		return structSize + stringSize
	}

	rtStructSize := int64(unsafe.Sizeof(*(data.token)))
	rtStringSize := int64(len(data.token.Token))
	return structSize + stringSize + rtStructSize + rtStringSize
```

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/f7ee09262991a3e783a5d792984b29cec285ccc0